js/core/css/examples.css

```diff
@@ -3,8 +3,11 @@ div.example-title {
     min-width: 7.5em;
     color: #b9ab2d;
 }
-div.example-title span {
-    text-transform: uppercase;
+div.example-title span:nth-child(1) {
+  text-transform: uppercase;
+}
+div.example-title span:nth-child(2) {
```

Comment:
This looks redundant... do we need this?

Reply:
Had to keep this because of base.css (I mainly translated this part so things work as they were working before)
https://github.com/w3c/tr-design/blob/f5285732ffcf13a7379c721758d7ea8baf8bf343/src/base.css#L659,L660

Reply:
I'm talking about the second `span:nth-child(2)` thing, I thank we can skip it.

For `.marker`, it seems tr-design prefers `.marker` over `.example-title`.

Reply:
`.marker` add `text-transform: uppercase`. So, that way, here `span:nth-child(1)` is redundant. So we can skip it but we need `span:nth-child(2)`. Earlier we added inline style for that.

Reply:
>Earlier we added inline style for that.

That's because we didn't have `nth-child(1)`. Now we have it, so we don't need explicit uppercasing. (It should implicitly get `text-transform: none`.)

Reply:
>.marker adds text-transform: uppercase.

Ah, I got it. Ignore the previous comment, sorry 😅